src/proto/play.proto

```diff
@@ -8,25 +8,24 @@ enum PlayName
     ShootOrChipPlay                    = 2;
     KickoffFriendlyPlay                = 3;
     OffensePlay                        = 4;
-    CornerKickPlay                     = 5;
-    ShootOrPassPlay                    = 6;
-    BallPlacementPlay                  = 7;
-    HaltPlay                           = 8;
-    EnemyFreeKickPlay                  = 9;
-    ExamplePlay                        = 10;
-    DefensePlay                        = 11;
-    EnemyBallPlacementPlay             = 12;
-    PenaltyKickEnemyPlay               = 13;
-    PenaltyKickPlay                    = 14;
-    FreeKickPlay                       = 15;
-    CreaseDefensePlay                  = 16;
-    StopPlay                           = 17;
-    KickoffEnemyPlay                   = 18;
-    AssignedTacticsPlay                = 19;
-    ScoringWithStaticDefendersPlay     = 20;
-    DribblingParcourPlay               = 21;
-    ScoringFromContestedPossessionPlay = 22;
-    PassEndurancePlay                  = 23;
+    ShootOrPassPlay                    = 5;
```

Comment:
you should reserve `5` and keep the existing proto numbers so that we don't break previous replays

Reply:
Updated